content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -194,6 +194,9 @@ kubectl get pods --show-labels
 JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}' \
  && kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=True"
 
+# Output decoded secrets without external tools
+kubectl get secret ${secret_name} -o go-template='{{range $k,$v := .data}}{{$k}}={{$v|base64decode}}{{"\n"}}{{end}}'
```

Comment:
BTW this would be more idiomatic:
`kubectl get secret my-secret -o go-template='{{range $k,$v := .data}}{{$k}}={{$v|base64decode}}{{"\n"}}{{end}}'`

Reply:
ah, sure.. shall I submit another PR?

Reply:
and I could improve formatting to work better for secret which have line-breaks
```
{{range $k,$v := .data}}{{"### "}}{{$k}}{{"\n"}}{{$v|base64decode}}{{"\n\n"}}{{end}}
```

Reply:
in fact, here we go: https://github.com/kubernetes/website/pull/26092